README.md

```diff
@@ -145,19 +141,27 @@ The following named groups can be matched from the output:
   * `file` - **[required]** the file to open. May be relative `cwd` or absolute. `(?<file> RE)`.
   * `line` - *[optional]* the line the error resides on. `(?<line> RE)`.
   * `col` - *[optional]* the column the error resides on. `(?<col> RE)`.
+  * `message` - *[optional]* Catch the humanized error message. `(?<message> RE)`.
 
-Since the regular expressions are written in a JSON file, backslashes must be escaped.
+Backslashes must be escaped, thus the double `\\` everywhere.
 
 The `file` should be relative the `cwd` specified. If no `cwd` has been specified, then
 the `file` should be relative the project root (e.g. the top most directory shown in the
 Atom Editor).
 
 If your build outputs multiple errors, all will be matched. Press `cmd-alt-g` (OS X) or `ctrl-alt-g` (Linux/Windows)
-to cycle through the errors (in the order they appear, first on stderr then on stdout).
+to cycle through the errors (in the order they appear, first on stderr then on stdout), or you can use the
+Atom Linter integration discussed in the next section.
 
 Often, the first error is the most interesting since other errors tend to be secondary faults caused by that first one.
 To jump to the first error you can use `cmd-alt-h` (OS X) or `shift-f4` (Linux/Windows) at any point to go to the first error.
 
+### Error matching and Atom Linter
+
+Install [Atom Linter](https://atom.io/packages/linter) and all your matched errors will listed in a neat panel.
+
+![Linter integration](https://noseglid.github.io/build-linter.png)
```

Comment:
ProTip: Upload this photo to github via new issue page and paste it here. Those links don't expire and are hosted on github